Load gmail js first

d-air1 opened the pull request:
## Description
Second attempt at pull request.

Commit:
This should finally get gmail.js to load before the page without being
blocked by chrome. It should also expose the gmail object simply by
using the "gmail" variable inside of your browsers console. No
congiguration needed courtesy of our secondary boiler plate code located
here "https://github.com/josteink/gmailjs-node-boilerplate".

Commit:
Recreate pull request.

d-air1 committed:
Merge branch 'master' into load-gmail-js-first

pranmar123 commented:
Open up the console and see a hello world to check if its working.  Make sure UBlock is disabled.

d-air1 commented:
Just wanted to clarify you're looking for hello followed by your email. The gmail object should already be created and it calls the get_email() function when saying hello.

AlecHolloway committed:
Update extension.js

The added code checks for a sense of urgency by scanning the text with specific keywords

pranmar123 committed:
Merge pull request #21 from pranmar123/sense_of_urgency

Sense of urgency
